eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/WorkOrderRest.java

```diff
@@ -70,6 +70,8 @@ public Response createWorkOrder(WorkOrder workOrder) {
 	public Response updateWorkOrder(WorkOrder workOrder) {
 		try {
 			assumeEquipmentMonoOrg(workOrder);
+			// We confirm the following on the front-end
+			workOrder.setConfirmedIncompleteChecklist(true);
```

Comment:
Wouldn't it be preferable to have the client send this as a parameter, and then just respect it? If another interface, even on EAM Light, updates a workOrder with open checklists, we would force this parameter. The safest option would be to fail in such case and force the client to be aware

Reply:
Done, see https://github.com/cern-eam/eam-light-frontend/pull/139/commits/d61d36ffbe6e98508498809e7882d30e3175226d.
Proposing closure of PR.